### PR TITLE
OSDOCS-12806: Corrected the toolbox image for 4.16+

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16+

Issue:
[OSDOCS-12806](https://issues.redhat.com/browse/OSDOCS-12806)

Preview:
* [About toolbox - Dedicated](https://85648--ocpdocs-pr.netlify.app/openshift-dedicated/latest/support/gathering-cluster-data.html#about-toolbox_gathering-cluster-data)
* [About toolbox - Core](https://85648--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#about-toolbox_gathering-cluster-data)
* [About toolbox - Rosa](https://85648--ocpdocs-pr.netlify.app/openshift-rosa/latest/support/gathering-cluster-data.html#about-toolbox_gathering-cluster-data)

- [x] SME has approved this change (@Timothee Ravier).
- [x] QE has approved this change (Mike Nguyen).

### CODE REVIEW

File: modules/support-starting-an-alternative-image-with-toolbox.adoc [hunk 1]
```diff
@@ -27,24 +33,19 @@ endif::openshift-origin[]
...
-----
-# vi ~/.toolboxrc
-----
+. Optional: If you need to use an alternative image instead of the default image, create a `.toolboxrc` file in the home directory for the root user ID, and specify the image metadata:
 +
 [source,text]
 ----
 REGISTRY=quay.io                <1>
 IMAGE=fedora/fedora:33-x86_64   <2>
 TOOLBOX_NAME=toolbox-fedora-33  <3>
```
Comment (mike-nguyen) [inline modules/support-starting-an-alternative-image-with-toolbox.adoc:42]: While we're in this document, can we change do the following  since fedora 33 is quite outdated.
```suggestion
IMAGE=fedora/fedora:latest   <2>
```

Comment (mike-nguyen) [inline modules/support-starting-an-alternative-image-with-toolbox.adoc:42]:
```suggestion
TOOLBOX_NAME=toolbox-fedora-latest  <3>
```

Comment (dfitzmau) [reply modules/support-starting-an-alternative-image-with-toolbox.adoc]: :heavy_check_mark:

Comment (dfitzmau) [reply modules/support-starting-an-alternative-image-with-toolbox.adoc]: :heavy_check_mark:

**Review by travier — APPROVED** [review]
/lgtm

File: modules/support-installing-packages-to-a-toolbox-container.adoc [hunk 2]
```diff
@@ -7,16 +7,17 @@
...
 
 ifdef::openshift-origin[]
-By default, running the `toolbox` command starts a container with the `quay.io/fedora/fedora:36` image. This image contains the most frequently used support tools. If you need to collect node-specific data that requires a support tool that is not part of the image, you can install additional packages.
+By default, running the `toolbox` command starts a container with the `quay.io/fedora/fedora` image. This image contains the most frequently used support tools. If you need to collect node-specific data that requires a support tool that is not part of the image, you can install additional packages.
 endif::openshift-origin[]
 
 .Prerequisites
 
 * You have accessed a node with the `oc debug node/<node_name>` command.
+* You can access your system as a user with root privileges. All the steps in the procedure commands require root privileges to run.
```
Comment (jeana-redhat) [inline modules/support-installing-packages-to-a-toolbox-container.adoc:20]: Redundant, especially since the cmd boxes have `#` prompts
```suggestion
* You can access your system as a user with root privileges.
```

File: modules/support-starting-an-alternative-image-with-toolbox.adoc [hunk 3]
```diff
@@ -7,16 +7,22 @@
...
 
 ifdef::openshift-origin[]
-By default, running the `toolbox` command starts a container with the `quay.io/fedora/fedora:36` image. You can start an alternative image by creating a `.toolboxrc` file and specifying the image to run.
+By default, running the `toolbox` command starts a container with the `quay.io/fedora/fedora` image. You can start an alternative image by creating a `.toolboxrc` file and specifying the image to run.
 endif::openshift-origin[]
 
 .Prerequisites
 
 * You have accessed a node with the `oc debug node/<node_name>` command.
+* You can access your system as a user with root privileges. All the steps in the procedure commands require root privileges to run.
```
Comment (jeana-redhat) [inline modules/support-starting-an-alternative-image-with-toolbox.adoc:25]:
```suggestion
* You can access your system as a user with root privileges.
```

File: modules/support-starting-an-alternative-image-with-toolbox.adoc [hunk 4]
```diff
@@ -27,24 +33,19 @@ endif::openshift-origin[]
...
-TOOLBOX_NAME=toolbox-fedora-33  <3>
+REGISTRY=quay.io             <1>
+IMAGE=fedora/fedora:latest   <2>
+TOOLBOX_NAME=toolbox-fedora-latest  <3>
 ----
-<1> Optional: Specify an alternative container registry.
+<1> This line is optional. Specify an alternative container registry.
 <2> Specify an alternative image to start.
-<3> Optional: Specify an alternative name for the toolbox container.
+<3> This line is optional. Specify an alternative name for the toolbox container.
```
Comment (jeana-redhat) [inline modules/support-starting-an-alternative-image-with-toolbox.adoc:46]: Should revert this change. `Optional:` is the standard we specify in the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks) and is used consistency in the repo (2000 hits vs under 100)
image  image

**Review by jeana-redhat — COMMENTED** [review]
I know this is existing content, but there should not be backticks in any assembly or module titles. You might consider fixing that while you are here, up to you :)

Otherwise, just a couple things here.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done
